src/pages/api/identities.ts

```diff
@@ -1,9 +1,9 @@
 import { HttpStatus } from "@fwl/web";
 
 import { Handler } from "../../@types/ApiPageHandler";
-import { addIdentityToUser } from "../../command/addIdentityToUser";
-import { removeIdentityFromUser } from "../../command/removeIdentityFromUser";
-import { withAPIPageLogger } from "../../middleware/withAPIPageLogger";
+import { addIdentityToUser } from "../../commands/addIdentityToUser";
+import { removeIdentityFromUser } from "../../commands/removeIdentityFromUser";
+import { withAPIPageLogger } from "../../middlewares/withAPIPageLogger";
```

Comment:
Can you use path alias here?

Reply:
OK fixed ccfa47e

Reply:
Hm I'm confused, all I can see are changes I made yesterday in your commit.

Reply:
Sorry, it was this commit instead cf9c211